debug/init.js

```diff
@@ -16,7 +16,11 @@ var cy, defaultSty, options;
           'source-arrow-shape': 'triangle-backcurve',
           'target-arrow-shape': 'triangle',
           'mid-target-arrow-shape': 'triangle',
-          'mid-source-arrow-shape': 'triangle-backcurve'
+          'mid-source-arrow-shape': 'triangle-backcurve',
+          'target-arrow-fill': 'hollow',
+          'source-arrow-fill': 'hollow',
+          'source-arrow-width': 2,
+          'target-arrow-width': '50%',
```

Comment:
Would you apply these new styles only to a subset of the edges?  The default style here should ideally show as many combinations of styles upfront.  

That way, anyone who looks at the debug page graph would immediately notice if any of these styles are broken.  E.g., you can see right away that both filled and hollow edges are OK.

Reply:
In my case, and more generally in SBGN in general, yes, only some edges are hollow, the rest is filled.

Here in the debug, you can see filled inside the "mid" arrows, and hollow at the "end" arrows. Then indeed, this test doesn't represent the full extent of all possibilities as it doesn't include the default "match-line", nor the usage of em units.
Do you want me to include them as well?

Reply:
Done in https://github.com/cytoscape/cytoscape.js/pull/3194/commits/69843f90fb10cd9cc7a47f00bc9026e1fd446e58

Reply:
Great.  Thanks